resources/prosody-plugins/token/util.lib.lua

```diff
@@ -30,11 +127,62 @@ local function _verify_token(token, appId, appSecret, disableRoomNameConstraints
 		return nil, "'room' claim is missing";
 	end
 
+    local audClaim = claims["aud"];
```

Comment:
in future please do not mix up adding new logic with moving things around especially if the commit msg says "this commit moves things to.."